agent/unit-test/softwarecontaineragent_unittest.cpp

```diff
@@ -53,12 +53,33 @@ class SoftwareContainerAgentTest: public ::testing::Test
     }
 };
 
-TEST_F(SoftwareContainerAgentTest, CreateContainerWithNoConf) {
-    ContainerID id = sca->createContainer("iejr-", "");
-    // This is actually only true if no other containers have been created
-    // before this one. Might need to be fixed somehow.
-    ASSERT_TRUE(id == 0);
+TEST_F(SoftwareContainerAgentTest, CreatAndCheckContainer) {
+	SoftwareContainer *container;
+	ContainerID id = sca->createContainer("iejr-", "");
+    bool retval = sca->checkContainer(id, container);
+    ASSERT_TRUE(retval == true);
+}
+
+TEST_F(SoftwareContainerAgentTest, DeleteContainer) {
+	SoftwareContainer *container;
+	ContainerID id = sca->createContainer("iejr-", "");
+	sca->deleteContainer(id);
+	bool retval = sca->checkContainer(id, container);
+	ASSERT_TRUE(retval == false);
 }
+/*
+ * Stress test written for creation + deletion
+ * It takes too much time, thus commented out
+ *
+TEST_F(SoftwareContainerAgentTest, StressCreateDeleteContainer) {
+	ContainerID id = 0;
+	for (auto i = 0 ; i < UINT32_MAX; i++) {
```

Comment:
Yes, another suite sounds easiest. What exactly is the easiest way to select/deselect and organize tests depends on the framework used I'd say, but both in the case of gtest and py.test I think it would be "suites". Let's move stress tests to a different separate update perhaps?